src/wasm/request.rs

```diff
@@ -180,6 +190,14 @@ impl RequestBuilder {
         self
     }
 
+    /// TODO
```

Comment:
This can probably be just a simple sentence, since the main docs use the non-wasm version.

Reply:
See https://github.com/seanmonstar/reqwest/pull/970